S03-operators/eqv.t

```diff
@@ -205,4 +205,17 @@ subtest 'Throws/lives in lazy cases' => {
     lives-ok    { (1…∞).Array eqv (1…3).Array }, 'Arrays, only one lazy';
 }
 
+# rakudo/issues/3346
+{
+    my class Null does Iterable does Iterator {
+        method iterator  { self }
+        method pull-one  { IterationEnd }
+    }
+
+    my $p = start { Null.new eqv Null.new };
+    await Promise.anyof($p, Promise.in(1));
```

Comment:
Please, make it 5 as was proposed. Sometimes on heavily loaded slow setups this might result in false negative. Bigger value is fine for a failing test. While passing test won't bother at all.

Reply:
As you wish.

Reply:
In that regard, why not going for 60 instead for the same reasons? 5 seconds may seem long, but once you turn on MoarVM's GC debugging to max, 60s are more like a blink of an eye.

Reply:
@niner It is a tradeoff. How many people do GC debugging? While there is an average limit for human's patience. 5-15 seconds range fits into the limit but 60 is definitely not. :) 

If we're gonna take things like this into account, we probably need to provide a default timeout value somehow which would depend on wether some heavy task is on or off. Then, say, `Promise.in($*TEST-TIMEOUT)` would handle such situations well enough.